keras_core/backend/jax/random.py

```diff
@@ -6,23 +6,30 @@
 from keras_core.random.seed_generator import make_default_seed
 
 
+def jax_draw_seed(seed):
+    if isinstance(seed, jax.Array):
+        return seed
+    else:
+        return draw_seed(seed)
```

Comment:
Shouldn't we always convert the seed to a jax PRNGKey in case the backend is JAX? I am just thinking about the advantage of the passing any other seed type

Reply:
There are no particular performance advantages (or any other advantages) to passing a PRNGKey over a JAX array. The PRNGKey class is syntactic sugar.

Reply:
Sorry for the confusion. I was referring to the other types (else branch in this case), like integer

Reply:
oh yeah the single integers are probably getting broadcast from `x -> [x, x]`, which isn't obviously any better or worse than `x -> [0, x]` which is what `PRNGKey` seems to do in practice, though it might do something different with non-default rng implementations?

Reply:
hmm actually passing an `int` into most of these functions is actually throwing errors!
`TypeError: JAX encountered invalid PRNG key data: expected key_data to have ndim, shape, and dtype attributes. Got 0`

Reply:
I'm going to add a fix to this PR